website/pages/en/_meta.js

```diff
@@ -3,19 +3,24 @@ export default {
   '---1': {
     type: 'separator',
   },
+  '###1': {
+    type: 'heading',
+    title: 'Overview',
+  },
   about: '',
   network: 'The Graph Network',
-  sunrise: 'Post-Sunrise Upgrade FAQ',
-  billing: '',
-  glossary: '',
   tokenomics: 'Tokenomics',
-  arbitrum: 'Scaling with Arbitrum',
+  contracts: '',
```

Comment:
I could be wrong, but I feel like Tokenomics should be under "The Graph Network". And maybe even Contracts?